src/modules/poolModule.ts

```diff
@@ -751,7 +763,10 @@ export class PoolModule implements BaseModule {
     let allTickLiquidities: TickLiquidity[] = [];
 
     while (hasNextPage) {
-      const response = await fetchTickLiquidityApi(this.sdk.BaseUrl, poolId, limit, offset);
+      const response = await fetchTickLiquidityApi(this.sdk.BaseUrl, poolId, limit, offset, {
+        ...this.sdk.customHeaders,
```

Comment:
Could you please confirm whether this will not throw if customHeader === undefined?